js/about/bookmarks.js

```diff
@@ -180,14 +180,28 @@ class BookmarksList extends ImmutableComponent {
   }
 }
 
+class SearchResults extends React.Component {
+  render () {
+    return (
+      <list className='bookmarkList'>
+      {
+        this.props.bookmarks.map((bookmark) => <BookmarkItem bookmark={bookmark} />)
+      }
+      </list>
+    )
+  }
+}
+
 class AboutBookmarks extends React.Component {
   constructor () {
     super()
     this.onChangeSelectedFolder = this.onChangeSelectedFolder.bind(this)
```

Comment:
When selected folder changes please clear out the current search term and show the folder contents instead